evm/build.gradle

```diff
@@ -30,6 +30,9 @@ jar {
   }
 }
 
+repositories {
+  maven { url "https://artifacts.consensys.net/public/maven/maven/" }
+}
```

Comment:
This should not be here, the root gradle should cover this.  If it doesn't it should be fixed there.